Sources/Functional_Remove.swift

```diff
@@ -65,7 +65,7 @@ extension ConnectionType {
 
     - parameter item: a `Persistable` item
     */
-    public func asyncRemove<Item: Persistable>(item: Item, queue: dispatch_queue_t = dispatch_get_main_queue(), completion: dispatch_block_t? = .None) {
+    public func asyncRemove<Item: Persistable>(_ item: Item, queue: DispatchQueue = DispatchQueue.main, completion: (()->())? = .none) {
```

Comment:
I think the preferred way must change like the direction of the wind... In Swift 2 I think it was Void -> (), but now that you have to wrap the params in () in Swift 3, maybe they've gone back to () instead of (Void)?

(I kinda hope so honestly)

Happy to change it to whatever though.